Source/r_things.c

```diff
@@ -702,6 +713,7 @@ void R_DrawPSprite (pspdef_t *psp)
   vis->x1 = x1 < 0 ? 0 : x1;
   vis->x2 = x2 >= viewwidth ? viewwidth-1 : x2;
   vis->scale = pspritescale;
+  vis->color = CR_DEFAULT;
```

Comment:
This shouldn't be strictly necessary, since `mobjflags2` never has the `MF2_COLOREDBLOOD` flag for a psprite?

Reply:
Removed in https://github.com/fabiangreffrath/woof/pull/213/commits/efa472c87bb97315821c4ae99c7844c850aa9af5